lodash.src.js

```diff
@@ -7325,25 +7325,17 @@
      * _.map(_.sortByAll(users, ['user', 'age']), _.values);
      * // => [['barney', 26], ['barney', 36], ['fred', 30], ['fred', 40]]
      */
-    function sortByAll() {
```

Comment:
Yeah, restParam is the best - I'm glad we figured out a way to get that
pattern abstracted

On Sun, Apr 12, 2015 at 12:41 AM, John-David Dalton <
notifications@github.com> wrote:

> In lodash.src.js
> <https://github.com/lodash/lodash/pull/1125#discussion_r28200638>:
>
> > @@ -7325,25 +7325,17 @@
> >       * _.map(_.sortByAll(users, ['user', 'age']), _.values);
> >       * // => [['barney', 26], ['barney', 36], ['fred', 30], ['fred', 40]]
> >       */
> > -    function sortByAll() {
>
> Super nice cleanup of this method. _.restParam is great for making things
> more readable!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lodash/lodash/pull/1125/files#r28200638>.
>